docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -5,7 +5,7 @@ sidebar_label: Managing credentials
 
 # Managing Credentials
 
-To manage the progression of freight from stage to stage, Kargo will often
+To manage the progression of `Freight` from `Stage` to `Stage`, Kargo will often
```

Comment:
I agree with the changes to uppercase here, but probably wouldn't monospace here.

```suggestion
To manage the progression of Freight from Stage to Stage, Kargo will often
```